internal/beef/beef.go

```diff
@@ -27,6 +27,19 @@ const (
 type TxData struct {
 	Transaction *bt.Tx
 	BumpIndex   *bt.VarInt
+	txID        string
+}
+
+func (td *TxData) Unmined() bool {
+	return td.BumpIndex == nil
```

Comment:
One suggestion. How about calling this `Mined()` or `IsMined()`

```
func (td *TxData) Mined() bool {
	return td.BumpIndex != nil
}
```

Then this would become simpler to read
```
	if !parent.Unmined() {
		*minedAncestors = append(*minedAncestors, parent)
		return nil
	}
```

Reply:
Right, done :)